src/checker/checks/any/invalidCreateOption.ts

```diff
@@ -8,8 +8,8 @@
  *   CREATE TABLE test;
  */
 
-import { IChecker } from "../interface";
-import { Create } from "../../barrel/statements";
+import { IChecker } from "../../interface";
+import { Create } from "../../../barrel/statements";
```

Comment:
This is hardcoded to the MySQL statements so we'll need to think of a way around this.

Reply:
I'm thinking the only real way is to split out all invalid*options into their own drivers which makes sense really. So do:
```
mysql/InvalidCreateOption.ts
import { Create } from "../../../barrel/mysql/statements";

public checker = Create();

```